src/Sql/Sql/help/Set-AzSqlInstance.md

```diff
@@ -182,7 +224,7 @@ Parameter Sets: SetInstanceFromInputParameters
 Aliases: InstanceName
 
 Required: True
-Position: 0
+Position: 1
```

Comment:
@johnpaulkee same comment about reversing the position of `-Name` and `-ResourceGroupName`

Reply:
Acked.